src/Akka.Persistence.Azure/TableEntities/PersistentJournalEntry.cs

```diff
@@ -77,14 +92,12 @@ public PersistentJournalEntry()
         ///     We store this as a separate field since we may pad the <see cref="RowKey" />
         ///     in order to get the rows for each partition to sort in descending order.
         /// </remarks>
-        public long SeqNo { get; set; }
+        public long SeqNo { get; }
 
         /// <summary>
         ///     Tags associated with this entry, if any
         /// </summary>
-        public string[] Tags { get; set; }
-
-        public DateTimeOffset Timestamp { get; set; }
+        public string[] Tags { get; }
```

Comment:
This property is the main reason why I opted not to use the built-in serializer. In the strongly typed json based serializer, this would be saved as a JSON array, while we traditionally have been saving it as a CSV. Trying to coerce the internal JSON serializer to serialize this as CSV would be a mess.